### DISCUSSION TITLE
Handeling Non UTF-8 examples? (Images, PDFs)

### DISCUSSION THREAD

**Discussion by ghost** [microcks/microcks#977] [Ideas]
How do we handle Non UTF-8 examples?

**Comment by lbroudoux** [comment]
Hey! As stated, we don't have a way to specify example values that cannot be integrated as UTF-8 encoded content into an OpenAPI spec file.

I see 2 ways of doing such a thing:
* Using a redirection as you did in https://github.com/orgs/microcks/discussions/927
* Maybe using `externalValue` as allowed by the spec but not supported in Microcks at the moment. As per the spec: `This provides the capability to reference examples that cannot easily be included in JSON or YAML documents.` We may add this feature later on.

What do you think?

**Reply by lbroudoux** [reply]
Also have some discussions on this topic here: https://github.com/microcks/microcks/issues/921
